src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.cpp

```diff
@@ -8,39 +8,61 @@
 #include "common/utils.hpp"
 #include "dnnl_extension_utils.h"
 #include "transformations/snippets/x64/op/brgemm_cpu.hpp"
+#include "openvino/op/matmul.hpp"
```

Comment:
Minor:
I didn't find the place in the code which needs this include. If it's extra include, could you remove please?

Reply:
Right, we don't need it anymore